libcontainer/selinux/selinux.go

```diff
@@ -58,16 +57,33 @@ func getSelinuxMountPoint() string {
 	}
 	selinuxfs = ""
 
-	mounts, err := mount.GetMounts()
+	f, err := os.Open("/proc/self/mountinfo")
 	if err != nil {
 		return selinuxfs
 	}
-	for _, mount := range mounts {
-		if mount.Fstype == "selinuxfs" {
-			selinuxfs = mount.Mountpoint
-			break
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		txt := scanner.Text()
+		sepIdx := strings.Index(txt, " - ")
+		if sepIdx == -1 {
+			return selinuxfs
```

Comment:
@crosbymichael Thanks! I referred to [cgroups.getCgroupMountsHelper()](https://github.com/opencontainers/runc/blob/3237cad790dcc0b5c1b3ec9f6efbdf3fd80d62fc/libcontainer/cgroups/utils.go#L129-L132) to parse `mountinfo`. Is it better to `continue` the loop here for robustness?